server/entity/MediaRequest.ts

```diff
@@ -767,7 +767,16 @@ export class MediaRequest {
         if (
           media[this.is4k ? 'status4k' : 'status'] === MediaStatus.AVAILABLE
         ) {
-          throw new Error('Media already available');
+          logger.warn('Media already available, marking request as APPROVED', {
```

Comment:
Maybe we should change the wording to "Media already exists in Radarr" since it may not actually be fully available?

Reply:
Sounds good!